### ISSUE TITLE
Pokeballs

### ISSUE THREAD

**Issue by Mercerenies** [EvanSkiStudios/Textile#8]
I'm adding craft-able pokeballs to Turtle's Troll. They're eggs with some custom NBT, and I can set a custom model ID as well. If you wanted to draw some item images of pokeballs to replace these special eggs, that would be pretty neat.

**Comment by EvanSkiStudios** [comment]
@Mercerenies 
What is the Minecraft item name you are using?
like with advanced tooltips you can see the "minecraft:(item)" below so for a stick its "minecraft:stick"

If you let me know what item it is i'll get to work on some sprites

**Comment by Mercerenies** [comment]
They're disguised eggs, so `minecraft:egg`

**Comment by EvanSkiStudios** [comment]
So when you give the player the egg or whatever, make sure the nbt for the egg contains this

> CustomModelData:(special number unique to this item)

so the regular pokeball can be 1
great ball 2
ultra 3

ect

**Comment by Mercerenies** [comment]
Alright, we can make the convention be:
* Pokeball - 1
* Great ball - 2
* Ultra ball - 3
* Master ball - 4
* Net ball - 5
* Dive ball - 6
* Dusk ball - 7

That's all of the ones I've programmed right now. Any you don't have time / feel like drawing we can just replace with the default pokeball image.

And yes, I did make master balls craftable. 100% chance of capturing any non-boss mob. It takes one redstone and eight netherite to craft one.
